versioned_docs/version-2.0.0/quickstart/node-express-mysql.md

```diff
@@ -0,0 +1,293 @@
+
+  Make API Calls using [Hoppscotch](https://hoppscotch.io), [Postman](https://postman.com) or cURL command. Keploy with capture those calls to generate the test-suites containing testcases and data mocks.
+
+  ```bash
+  curl -X POST http://localhost:3000/items \
+     -H "Content-Type: application/json" \
+     -d '{"name": "Sample Item", "description": "This is a sample description"}'
+
+  ```
+
+  Here's a peek of what you get:
+
+  ```bash
+  Item added with ID!
+  ```
+
+  🎉 Woohoo! Give yourself a pat on the back! With that simple spell, you've conjured up a test case with a mock! Explore the **Keploy directory** and you'll discover your handiwork in `test-1.yml` and `mocks.yml`.
+
+  <img src="/docs/img/testcase-node.png" alt="Sample Keploy Test Result Node MongoDB" width="100%" style={{ borderRadius: '5px' }}/>
+
+  Now, the real fun begins. Let's weave more spells!
+
+  🚀 Follow the URL road...!
+
+  ```bash
+  curl http://localhost:3000/items
+  ```
+
+  Did you spot the new test and mock scrolls in your project library? Awesome! 👏
+
+  ### Run Tests 🏁
+
+  Ready to put your spells to the test?
+
+  ```bash
+  sudo -E env PATH=$PATH keploy test -c "node ./server.js" --delay 10
+  ```
+
+  Worry not, just add the ever-changing fields (like our **ts** here) to the **noise parameter** to **dodge those assertions**.
+
+  <img src="/docs/img/testrun-node-pass.png" alt="Sample Keploy Test Result Node sql" width="100%" style={{ borderRadius: '5px' }}/>
+
+  ## Wrapping it up 🎉
+
+  Congrats on the journey so far! You've seen Keploy's power, flexed your coding muscles, and had a bit of fun too! Now, go out there and keep exploring, innovating, and creating! Remember, with the right tools and a sprinkle of fun, anything's possible.😊🚀
+
+  Happy coding! ✨👩‍💻👨‍💻✨
+     </details>
+   </details>
+    <br/>
+
+- <details>
+     <summary><img src="/docs/img/os/macos.png" alt="MacOS" width="3%" /> MacOs </summary>
+
+  Dive straight in, but first, give **Colima** a gentle nudge with (`colima start`). Let's make sure it's awake and ready for action!
+
+  #### Add alias for Keploy 🐰:
+
+  For the sake of convenience (and a bit of Mac magic 🪄), let's set up a shortcut for Keploy:
+
+  ```bash
+  alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v "$(pwd)":/files -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
+  ```
+
+  ### Lights, Camera, Record! 🎥
+
+  Fire up the application and mysql instance with Keploy. Keep an eye on the two key flags:
+  `-c`: Command to run the app (e.g., `docker compose up`).
+
+  `--containerName`: The container name in the `docker-compose.yml` for traffic interception.
+
+  ```sh
+  keploy record -c "docker compose up" --containerName "nodesqlapp"
```

Comment:
https://github.com/Sonichigo/samples-typescript/blob/main/js-mysql/docker-compose.yaml

In the docker-compose.yml file the containerName is not present. there this method won't work unless add manually by user following the guide.